JS/models/ClassificationLevel.js

```diff
@@ -0,0 +1,8 @@
+const ClassificationLevel  = {
```

Comment:
Please use CAPITAL_LETTER syntax for constants.

Reply:
Done